src/test/java/codeu/service/BBCodeParserTest.java

```diff
@@ -0,0 +1,64 @@
+package codeu.service;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class BBCodeParserTest {
+
+  @Test
+  public void testParse_Empty() {
```

Comment:
Google recommends that the test names describe what you expect.  So something like:

public void testParse_ParsingEmptyStringReturnsEmptyString

A quick Google search shows an article discussing something like this:

https://dzone.com/articles/7-popular-unit-test-naming

According to that article, you might do:

testParse_emptyString_returnsEmptyString.

However, I notice that the existing tests don't really include what's expected, so I leave it up to you whether you want to be consistent with the existing tests or be more descriptive.

Reply:
I think the more descriptive version is better, I'll rename the tests.